core/src/main/kotlin/ch/ergon/dope/resolvable/expression/unaliased/type/Primitive.kt

```diff
@@ -49,13 +49,15 @@ class Primitive<T : ValidType> : TypeExpression<T> {
     }
 }
 
+fun TypeExpression<StringType>.isBlank(): Boolean = toQueryString().drop(1).dropLast(1).isBlank()
```

Comment:
What is this logic?

Reply:
A typeExpression always has a " ate the start and end of its queryString, so to check if it is blank i need to remove them